products/compute/terraform.yaml

```diff
@@ -32,6 +32,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "internal_with_gce_endpoint"
         vars:
           address_name: "my-internal-address-"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
I think we can `skip_docs` on this one- it's basically identical to the other example except one field has been changed. I'm not sure that really provides much value to users to have both.

Reply:
It is true the new option is basically identical, but it is currently only for beta, and I wanted to make sure the option works. We could catch the error if it is accidentally slipped into ga, for example.

Reply:
I'm not sure what you mean by this. My suggestion was to leave the test in, but just to remove it from the docs.

Reply:
Sorry I misunderstood your suggestion. The `skip_docs` now has been added